pkg/diff/schema_migration_plan_test.go

```diff
@@ -1118,6 +1118,92 @@ var (
 				},
 			},
 		},
+		{
```

Comment:
These tests are a little unnecessary. It's a bit confusing but `schema_migration_plan_test.go` is really only for testing how the migrations are done, e.g., indexes added using `CONCURRENTLY`. 

In the case of extensions, we really don't need to check how it's done, since there's only one way of enabling/disabling, which you validated in the acceptance tests you wrote.

Sorry! My FUD around making these tests too big (the schema_migration_plan_test.go), is they're annoying to update if we ever update the internal structure of the schema objects.